pkg/pxc/backup/scheduled.go

```diff
@@ -98,4 +96,28 @@ func (bcp *Backup) scheduledJob(spec *api.PXCScheduledBackupSchedule, strg *api.
 			},
 		},
 	}
+
+	if strg.Resources != nil {
+		job.Template.Spec.Containers[0].Resources = *strg.Resources
+	}
+	if strg.Affinity != nil {
+		job.Template.Spec.Affinity = strg.Affinity
+	}
```

Comment:
no need in this check - if it's nil you gonna assign a nil to the variable that is nil.